docs/specs/clients/notify/client-sdk-api.md

```diff
@@ -40,24 +40,45 @@ abstract class Client {
   // get all notifications for a subscription
   public abstract getNotificationHistory(params: {
     topic: string,
-  }): Promise<Record<number, NotifyNotificationRecord>>
+    // Default 10, max 50
+    limit?: number,
+    startingAfter?: string,
+    // Default false
+    unreadFirst?: string,
+  }): Promise<{
+    notifications: Record<number, NotifyNotificationRecord>,
```

Comment:
I think should be `NotifiyNotification[]`, since we get an array back from notify server

Reply:
I'm OK with this